### PR TITLE
Add CSS

### PR DESCRIPTION
**Pull request by RoelN**
This will add the CSS generated from the font, and will replace the two spots where "naked" CSS was shown.

I considered using a full-blown formatter, but decided that I could get away with some basic formatting in my own code.

### CODE REVIEW

File: src/components/report/FeatureControls.js [hunk 1]
```diff
@@ -49,8 +51,16 @@ export default {
 				if (state === null) continue;
 				styles += `${glue} "${feature.tag}" ${state ? "1" : "0"}`;
 				glue = ",";
+				// Poor man's code formatting
+				if (++counter % maxProps === 0) {
+					glue = `,\n                      `;
+				}
+			}
+			if (styles) {
+				return `font-feature-settings:${styles};`;
```
Comment (pascalw) [inline src/components/report/FeatureControls.js:60]: Could this be moved to the engine perhaps? (Could be a separate PR)

File: src/components/report/VariableControls.js [hunk 2]
```diff
@@ -56,11 +61,21 @@ export default {
...
 			for (const axis of Object.values(this.axes)) {
 				styles += `${glue} "${axis.id}" ${axis.current}`;
 				glue = ",";
+				// Poor man's code formatting
+				if (++counter % maxProps === 0) {
+					glue = `,\n                        `;
+				}
+			}
+			if (styles) {
+				return `font-variation-settings:${styles};`;
```
Comment (pascalw) [inline src/components/report/VariableControls.js:75]: There seems to be quite some duplication between this and `FeatureControls`. Is there an opportunity to extract something?

**Review by pascalw — APPROVED** [review]
Nice, looks really cool too!

File: src/main.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import Vue from "vue";
 import App from "./App.vue";
 import "./filters";
+import "prismjs";
+import "prismjs/themes/prism-tomorrow.css";
```
Comment (pascalw) [inline src/main.js:5]: 🤩 Looks really great. Tomorrow night is a great color theme!

File: src/components/report/FeatureControls.js (hunk 1 above)
Comment (RoelN) [reply src/components/report/FeatureControls.js:60]: Perhaps, but the engine currently looks only at the font contents and returns generic CSS for that. This specific code here is dynamic: it needs to change in response to the user playing around with on/off/indeterminate buttons.

So for now I feel it belongs here, but maybe when improving the engine's CSS generation, we can account for this kind of dynamic CSS creation and move it.

File: src/components/report/VariableControls.js (hunk 2 above)
Comment (RoelN) [reply src/components/report/VariableControls.js:75]: They are indeed quite similar, but have quite a few subtle differences (data structure, CSS, indentation, number of props per line). I _think_ it would make it more complex to have one function to handle them both, than have these separate with some minor duplication.

I don't think these functions will be extended in the future, but if they do, look for opportunities  then?

File: src/main.js (hunk 3 above)
Comment (RoelN) [reply src/main.js:5]: It is, right?!
